tests/test_envtpl.py

```diff
@@ -119,7 +121,22 @@ def test_unicode_input(self):
         expected = u'''
 åäö
 '''
-        self.assertEquals(envtpl._render_string(string, {}, jinja2.StrictUndefined), expected)
+        self.assertEquals(envtpl._render_string(string, {}, jinja2.StrictUndefined,
+                          '{{,}}'), expected)
+
+    def test_custom_var_delimiter(self):
```

Comment:
Could you add tests to verify that we don't allow bad delimiters like `,`, `{{,`, etc.